src/Dynamo.All.sln

```diff
@@ -85,7 +85,7 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "DynamoCoreWpfTests", "..\te
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "IronPythonTests", "..\test\Libraries\DynamoPythonTests\IronPythonTests.csproj", "{773988FE-EDF6-45CB-A63F-482955EB3553}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "DSIronPython", "Libraries\DSIronPython\DSIronPython.csproj", "{9EEF4F42-6B3B-4358-9A8A-C2701539A822}"
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "DSCPython", "Libraries\DSCPython\DSCPython.csproj", "{9EEF4F42-6B3B-4358-9A8A-C2701539A822}"
```

Comment:
why was this project renamed instead of a new project being added?

Reply:
@Dewb Looks like the new project was created by copying the file of the old project.  This resulted in two projects with the same GUID, which confuses Visual Studio.  You need to manually change the `<ProjectGuid>` in the new project.

Reply:
Yes, the DSIronPython project entry is also present in line 228. Both the DSIronPython and DSCPython projects have the same GUID. I will change it to a different id.